torch/csrc/distributed/autograd/context/dist_autograd_container.cpp

```diff
@@ -78,6 +78,10 @@ DistAutogradContext& DistAutogradContainer::getOrCreateContext(
   return context;
 }
 
+int64_t DistAutogradContainer::getWorkerID() const {
```

Comment:
whoops, this should be `int16_t`. Will change after rebasing and fixing merge conflicts

Reply:
Shouldn't it be `rpc::worker_id_t` instead?

Reply:
Yes, would be good to replace `int16_t` with `worker_id_t` in the relevant places. Also, minor nit, lets call this `getWorkerId` similar to RpcAgent.